### ISSUE TITLE
How can I organise large schemas?

### ISSUE THREAD

**Issue by muku2211** [ent/ent#3961]
I'm building an ent schema that spans 5 different Postgres DB schemas.

Each DB schema has around 15 tables, and can be inter-related, which is why I'm using the same ent schema for all 5 DB schemas.

But that means around ~50 files in the `ent/schema` folder.

What's the best way to organise these files?

**Comment by Sinongamess** [comment]
I have the same issue, managing these files is a terrible experience. Have you found any solutions?

**Comment by muku2211** [comment]
Nope, haven't found any solutions yet. For now, I'm keeping all files in the same schema folder.
